preprod.healdata.org/manifests/hatchery/hatchery.json

```diff
@@ -0,0 +1,129 @@
+{
+  "user-namespace": "jupyter-pods-qa-heal",
+  "sub-dir": "/lw-workspace",
+  "user-volume-size": "10Gi",
+  "sidecar": {
+    "cpu-limit": "1.0",
+    "memory-limit": "256Mi",
+    "image": "quay.io/cdis/gen3fuse-sidecar:2021.03",
+    "env": {
+      "NAMESPACE": "qa-heal",
```

Comment:
@xritter1 I'm assuming I should find/replace just about everywhere in the automated changes and replace `qa` with `preprod`?